### ISSUE TITLE
Short Questions Thread

### ISSUE THREAD

**Issue by martin2250** [martin2250/GrblHeightProbe2#29]
Since some people didn't want to create their own issues for short questions I've created this permanent issue. You can also use it for discussions on the topic.

**Comment by momiccioli** [comment]
Where is the executable?  I downloaded the zip but there is no exe.

**Comment by martin2250** [comment]
https://github.com/martin2250/GrblHeightProbe2/releases
this is also linked to from the readme and the youtube video.

**Comment by momiccioli** [comment]
Found it and tried it.  Will this run with TinyG controller?  It connects but the commands don't get executed.  It would be great to have this and the Gcode sender all in one package.

**Comment by martin2250** [comment]
No, only grbl is supported. a while ago I started working on an integrated sender/autoleveller (with support for marlin/tinyg) but I just don't have the time to work on it.

**Comment by momiccioli** [comment]
Martin,

Thanks for the update, I know how it is thank you.
